### PR TITLE
Vishal/adding get transaction result

### PR DESCRIPTION
**Pull request by vishalchangraniaxiom**
## Description
Modifying the **execution** api:
1. Renaming GetEventsForBlockIDTransactionID to GetTransactionResult
2. The response for GetTransactionResult includes both the events and the transaction error if any.
The transaction error is included in the Access API GetTransactionResult

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: protobuf/flow/execution/execution.proto [hunk 1]
```diff
@@ -19,15 +19,17 @@ service ExecutionAPI {
     // Scripts
 
     // ExecuteScriptAtBlockID executes a ready-only Cadence script against the execution state at the block with the given ID.
-    rpc ExecuteScriptAtBlockID (ExecuteScriptAtBlockIDRequest) returns (ExecuteScriptResponse);
+    rpc ExecuteScriptAtBlockID (ExecuteScriptAtBlockIDRequest) returns (ExecuteScriptAtBlockIDResponse);
```
Comment (vishalchangraniaxiom) [inline protobuf/flow/execution/execution.proto:22]: There is only one Script related API unlike the access api which has several. Hence, making the response name more specific.

File: protobuf/flow/execution/execution.proto [hunk 2]
```diff
@@ -19,15 +19,17 @@ service ExecutionAPI {
...
 
     // Events
 
     // GetEventsForBlockIDs retrieves events for all the specified block IDs that have the given type
-    rpc GetEventsForBlockIDs (GetEventsForBlockIDsRequest) returns (EventsResponse);
+    rpc GetEventsForBlockIDs (GetEventsForBlockIDsRequest) returns (GetEventsForBlockIDsResponse);
```
Comment (vishalchangraniaxiom) [inline protobuf/flow/execution/execution.proto:27]: With the rename of `GetEventsForBlockIDTransactionID`, there only remains one Event related API. Renaming the response to be more specific

File: protobuf/flow/execution/execution.proto [hunk 3]
```diff
@@ -74,7 +76,15 @@ message GetEventsForBlockIDsRequest {
...
-message GetEventsForBlockIDTransactionIDRequest {
+// Transactions
+
+message GetTransactionResultRequest {
     bytes block_id = 1;
     bytes transaction_id = 2;
 }
+
+message GetTransactionResultResponse {
+    uint32 status_code = 2;
```
Comment (vishalchangraniaxiom) [inline protobuf/flow/execution/execution.proto:87]: Including **status_code** and **error** in the response is the main reason for the change.

File: protobuf/flow/execution/execution.proto [hunk 4]
```diff
@@ -19,15 +19,17 @@ service ExecutionAPI {
...
     // GetEventsForBlockIDs retrieves events for all the specified block IDs that have the given type
-    rpc GetEventsForBlockIDs (GetEventsForBlockIDsRequest) returns (EventsResponse);
+    rpc GetEventsForBlockIDs (GetEventsForBlockIDsRequest) returns (GetEventsForBlockIDsResponse);
 
-    // GetEventsForBlockIDAndTransactionID retrieves the events emitted by a transaction in a specific block.
-    rpc GetEventsForBlockIDTransactionID (GetEventsForBlockIDTransactionIDRequest) returns (EventsResponse);
+    // Transaction
+
+    // GetTransactionResult gets the result of a transaction.
+    rpc GetTransactionResult (GetTransactionResultRequest) returns (GetTransactionResultResponse);
```
Comment (psiemens) [inline protobuf/flow/execution/execution.proto:32]: 👍

File: protobuf/flow/execution/execution.proto (hunk 3 above)
Comment (psiemens) [inline protobuf/flow/execution/execution.proto:87]:
```suggestion
    uint32 status_code = 1;
```

These fields should start with `1`

**Pull request merged**